src/rpdk/core/build_image.py

```diff
@@ -0,0 +1,59 @@
+"""This sub command will build an image for the executable
+
+Projects can be created via the 'init' sub command.
+"""
+import logging
+
+import docker
+from docker.errors import APIError, BuildError
+
+from .exceptions import DownstreamError
+from .project import Project
+
+LOG = logging.getLogger(__name__)
+
+
+def build_image(_args):
+    project = Project()
+    project.load()
+
+    config = project.generate_image_build_config()
+
+    if _args.executable:
+        executable_name = _args.executable
+    else:
+        executable_name = config["executable_name"]
+
+    image_name = (
+        _args.image_name if _args.image_name else "-".join(project.type_info).lower()
+    )
+
+    docker_client = docker.from_env()
+    LOG.warning("Creating image with name '%s'", image_name)
```

Comment:
why it's `warning`? not `info`?

Reply:
I wanted to print something to the terminal at all times so that the developer knows something is happening